snuba/settings/__init__.py

```diff
@@ -227,9 +228,13 @@ class RedisClusters(TypedDict):
 COLUMN_SPLIT_MAX_LIMIT = 1000
 COLUMN_SPLIT_MAX_RESULTS = 5000
 
-# The migration groups that can be skipped are listed in OPTIONAL_GROUPS.
-# Migrations for skipped groups will not be run.
-SKIPPED_MIGRATION_GROUPS: Set[str] = {"querylog", "profiles", "functions"}
+# Migrations in skipped groups will not be run
+SKIPPED_MIGRATION_GROUPS: Set[str] = {
```

Comment:
I think access_logs needs to be on the OPTIONAL_GROUPS list for skipping to be enabled on that group